### PR TITLE
Fix field margins

### LINKED ISSUE #215: Field item margins
The last `field__item` or the last nested field shouldn't have a `margin-bottom` if it is not wrapped in `field__group`.

Also, `field__group` should have a `margin-bottom` of -1 * the `margin-bottom` of the inner `field__items`.

_Originally posted by @bameyrick in https://github.com/ONSdigital/pattern-library-v2/issues/35#issuecomment-466001706_

### PR DESCRIPTION
**Pull request by bameyrick**
Will resolve #215

### CODE REVIEW

**Comment by bameyrick** [comment]
[Issue #215: Field item margins](https://trello.com/c/ALp7K07v/171-issue-215-field-item-margins)

File: src/components/field/_field.scss [hunk 1]
```diff
@@ -17,6 +17,7 @@
 
   &__group {
     @extend .u-cf;
+    margin-bottom: -1rem;
```
Comment (jrbarnes9) [inline src/components/field/_field.scss]: The clearfix above already seems to being doing this job?  There are now spacing issues caused by this negative margin...
screenshot 2019-02-21 at 14 31 35

Comment (bameyrick) [reply src/components/field/_field.scss]: Hmm the clearfix didn't appear to have the effect of preventing the bottom margin on the date input. I'll take a deeper look into it.

Comment (bameyrick) [reply src/components/field/_field.scss]: Having had a look this behaviour is correct. Currently field doesn't actually apply any spacing between itself and another field (see https://sdc-prototypes.netlify.com/prototypes/example/ for an example). In runner this is achieved by wrapping each field in a custom `answer` class. 

We should maybe discuss the this margin being applied to field themselves outside of this PR.

Comment (bameyrick) [reply src/components/field/_field.scss]: I've began a discussion about this on the forum issue for field: https://github.com/ONSdigital/pattern-library-v2/issues/35#issuecomment-466045681

**Pull request merged**